src/components/DocRoot/index.js

```diff
@@ -103,6 +103,10 @@ export default function DocRoot() {
               New Relic Insights operates on a push model.
             </li>
 
+            <li><NavLink className="doc-section" to="/docs/registry/otlp">OpenTelemetry (OTLP)</NavLink>.
```

Comment:
To be even less ambiguous (people are already constantly confused), could we make this "OpenTelemetry Protocol (OTLP)"?